romana-install/roles/provision/create-aws-hosts/templates/aws_cf_small_cluster.json

```diff
@@ -1052,27 +1052,19 @@ under the License.
   "Parameters": {
     "ControllerInstanceType": {
       "Description": "EC2 instance type for Controller Node",
-      "Type": "String",
-      "Default": "t2.large",
-      "ConstraintDescription": "Must be a valid EC2 instance type."
```

Comment:
We don't want the constraint description anymore?

We might also consider adding something like this:

    "AllowedValues" : ["t1.micro", "m1.small", "m1.large", ... ],